js/controller/AppController.js

```diff
@@ -24,6 +24,10 @@ app.controller('AppController', function ($scope, $location, $http, $route, $log
     $rootScope.sidebar = {
         show: false
     };
+    $rootScope.navibar = {
+        show: true
```

Comment:
They have a different approach - They are using CSS to move (translate) the main container (right one) over the navigation pane :)